.github/workflows/build-and-test.yml

```diff
@@ -4,9 +4,9 @@
 name: Build and Test
 on:
   push:
-    branches: ["master"]
+    branches: ["main", "master"]
```

Comment:
As we only have a main branch, couldn't master just be replaced with it, rather than adding main to this array?